### DISCUSSION TITLE
inav targets

### DISCUSSION THREAD

**Discussion by rhoads6257** [iNavFlight/inav#8489] [Q&A]
When will iNav release a target for Speedybee f405 v3

**Comment by stronnag** [comment]
It is in master, so INAV 6.0
You can build it yourself now

**Reply by rhoads6257** [reply]
Thank You! That was the fix!

**Comment by DzikuVx** [comment]
It's available for 5.1 in Configurator already

Paweł Spychalski
________________________________
